README.md

```diff
@@ -5,6 +5,38 @@
 [![@latest](https://img.shields.io/npm/v/octoherd-script-download-files.svg)](https://www.npmjs.com/package/octoherd-script-download-files)
 [![Build Status](https://github.com/stefanbuck/octoherd-script-download-files/workflows/Test/badge.svg)](https://github.com/stefanbuck/octoherd-script-download-files/actions?query=workflow%3ATest+branch%3Amain)
 
+## Usage
+
+Minimal usage
+
+```js
+npx octoherd-script-download-files \
+  --source README.md \
+  --target ./out
+```
+
+Pass all options as CLI flags to avoid user prompts
+
+```js
+npx octoherd-script-download-files \
+  -T ghp_0123456789abcdefghjklmnopqrstuvwxyzA \
+  -R "stefanbuck/*" \
+  --ignoreArchived undefined \
```

Comment:
@gr2m this should be `--ignore-archived` or `--no-ignore-archived`, but the project scaffolder generate the line above. I'll have a look and submit a fix for the scaffolder.

Reply:
https://github.com/octoherd/create-octoherd-script/pull/71